Services/COPage/xsl/page.xsl

```diff
@@ -4376,6 +4376,24 @@
 	</xsl:if>
 </xsl:template>
 
+<!-- PRGActionNote -->
+<xsl:template match="PRGActionNote">
+	<xsl:if test="$mode = 'edit'">
+		<div class="copg-content-placeholder-prg-note il-prg-actionnote-container">
+			<div>
+				<img class="icon prg large" src="./templates/default/images/standard/icon_prg.svg" alt="Study Programme Action Note" />
+			</div>
+			<div>Study Programme Action Note</div>
+		</div>
+	</xsl:if>
+	<xsl:if test="$mode != 'edit'">
+		<div class="il-prg-actionnote-container">
+			<Launcher>
+				[[[PRG_ACTION_NOTE]]]
+			</Launcher>
+		</div>
+	</xsl:if>
+</xsl:template>
```

Comment:
@lukastocker The text "Study Programme Action Note" will not be translated here. Since you are already using the `modifyPageContentPostXsl()` approach you could reduce the XSL part to:
```
<xsl:template match="PRGActionNote">
    [[[PRG_ACTION_NOTE]]]
</xsl:template>
```

And handle the different edit/presentation rendering completely in `modifyPageContentPostXsl()` and use $DIC->language().